src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -30,5 +30,10 @@ We recommend using the following best practices when developing themes:
    ```xml
    <a href="#"><?= __('Click to download'); ?></a>
    ```
+1. By following the standard conventions, you give your code a professional look while making it easier to read. Ensure that your Magento coding standards are based on PSR coding standards.
+1. When inheriting Blank or Luma themes, Make use of the mobile first approach that is already been followeed in these themes.
+1. Avoid using redundant or duplicate code, which can be hard to maintain. Being it for styling (CSS, LESS) or developemnt (PHP), Instead of copy and paste of same code, create a single class or method and reference it when needed. Be sure to reuse code as much as possible.
```

Comment:
> 1. Avoid using redundant or duplicate code, which can be hard to maintain. Being it for styling (CSS, LESS) or developemnt (PHP), Instead of copy and paste of same code, create a single class or method and reference it when needed.

This part the author got from [Complete Guide on Magento 2 Development Best Practices ](https://www.icecubedigital.com/blog/complete-guide-on-magento-2-development-best-practices/). 

<img width="862" alt="6625-01" src="https://user-images.githubusercontent.com/13585327/74384756-af834580-4dfa-11ea-89e2-f7636f62432d.png">

Please use the same text formatting.